sample_data_config/twitter-ads-api/twitter-ads-api.config.yaml

```diff
@@ -1,7 +1,36 @@
 twitterAdsApi:
-  - dataPipelineId: twitter_ads_api_campaign_elife_journal
+  # - dataPipelineId: twitter_ads_api_campaign_elife_journal
+  #   source:
+  #     resource: '/11/accounts/18ce53up4ko/campaigns'
+  #     secrets:
+  #       parametersFromFile:
+  #         - parameterName: api_key
+  #           filePathEnvName: TWITTER_API_KEY_FILE_PATH
+  #         - parameterName: api_secret
+  #           filePathEnvName: TWITTER_API_SECRET_FILE_PATH
+  #         - parameterName: access_token
+  #           filePathEnvName: TWITTER_ACCESS_TOKEN_FILE_PATH
+  #         - parameterName: access_token_secret
+  #           filePathEnvName: TWITTER_ACCESS_TOKEN_SECRET_FILE_PATH
+  #   target:
+  #     projectName: 'elife-data-pipeline'
+  #     datasetName: '{ENV}'
+  #     tableName: twitter_campaign_details_elife_journal
+
+  - dataPipelineId: twitter_ads_api_campaign_reach_elife_journal
     source:
-      resource: '/11/accounts/18ce53up4ko/campaigns'
+      resource: '/11/stats/accounts/18ce53up4ko/reach/campaigns'
+      paramValueFromBigQuery:
+        projectName: 'elife-data-pipeline'
+        sqlQuery: |-
+          SELECT DISTINCT t_data.id
+          FROM `elife-data-pipeline.hc_dev.v_latest_twitter_campaign_details_elife_journal`
+          JOIN UNNEST(data) AS t_data
+          WHERE t_data.id IN ('git3k', 'gwdrg')
+      requiredParams:
+        paramNameForBqValue: campaign_ids
```

Comment:
minor / comment:
In general I would maybe try to avoid abbreviations. Sometimes it may not be clear for someone unfamiliar (although I guess in this case it is). The other argument would be consistency. So far all other references to "BigQuery" in the config seem to be `BigQuery` not `Bq`.

`parameter` is mostly spelled out, except for `params` (that may have been a mistake). I guess we used `params` because that is the name of the parameter within the requests library. Maybe we should change it? Perhaps even to make clear that these are query parameters passed in via the URL.

Here, what do you think of changing the name of the container to something like `requiredParameterNameFor`, then the fields could be `bigQueryValue`, `startTime` and `endTime`. Maybe we could also drop the `required` from the name? Maybe `apiQueryParameterNameFor` (not sure if that is too long or is any clearer)